### DISCUSSION TITLE
学习硬件开发首先需要掌握什么知识？

### DISCUSSION THREAD

**Discussion by sky41** [peng-zhihui/ElectronBot#258] [General]
我是一名计科同学，对硬件开发有很大的兴趣在大学里面主要研究是软件开发，想去研究一下如何开发硬件，希望各位大佬同学们给一些建议，谢谢！

**Comment by ZK-y53** [comment]
模拟电路、数字电路，protues仿真，EDA设计......先学这些吧
















在 2024-05-18 15:08:28，"skywide" ***@***.***> 写道：

我是一名计科同学，对硬件开发有很大的兴趣在大学里面主要研究是软件开发，想去研究一下如何开发硬件，希望各位大佬同学们给一些建议，谢谢！
